device-types/Dell/VxRail-P670F.yaml

```diff
@@ -0,0 +1,88 @@
+---
+manufacturer: Dell
+model: VxRail P670F
+slug: dell-vxrail-p670f
+u_height: 2
+is_full_depth: true
+airflow: front-to-rear
+weight: 35.2
+weight_unit: kg
+front_image: true
+rear_image: true
+comments: |
+  [Dell VxRail installation](https://dl.dell.com/content/manual44265077-dell-vxrail-p670f-p670n-v670f-and-s670-installation-and-service-manual.pdf?language=en-us)
+  [Dell VxRail datasheet](https://dl.dell.com/content/manual43304433-dell-vxrail-p670f-p670n-v670f-and-s670-technical-specifications.pdf?language=en-us)
+power-ports:
+  - name: Power 1
+    type: iec-60320-c14
+  - name: Power 2
+    type: iec-60320-c14
+interfaces:
+  - name: Gig-E 1
+    type: 1000base-t
+  - name: Gig-E 2
+    type: 1000base-t
+  - name: iDRAC
+    type: 1000base-t
+    mgmt_only: true
+module-bays:
+  - name: AHCI controller in SL 6
+    position: AHCI.SL.6-1
+    label: BOSS-S2
+  - name: Backplane 1 on Connector 0 of Storage Controller in SL 3
+    position: Internal.0-1:NonRAID.SL.3-1
+    label: BP15G+EXP 0:1
+  - name: CPU 1
+    position: CPU.Socket.1
+    label: Intel(R) Xeon(R) Gold 6342 CPU @ 2.80GHz
+  - name: CPU 2
+    position: CPU.Socket.2
+    label: Intel(R) Xeon(R) Gold 6342 CPU @ 2.80GHz
+  - name: DIMM A1
+    position: DIMM.Socket.A1
+    label: DDR4 DIMM
+  - name: DIMM A2
+    position: DIMM.Socket.A2
+    label: DDR4 DIMM
+  - name: DIMM A3
+    position: DIMM.Socket.A3
+    label: DDR4 DIMM
+  - name: DIMM A4
+    position: DIMM.Socket.A4
+    label: DDR4 DIMM
+  - name: DIMM A5
+    position: DIMM.Socket.A5
+    label: DDR4 DIMM
+  - name: DIMM A6
+    position: DIMM.Socket.A6
+    label: DDR4 DIMM
+  - name: DIMM A7
+    position: DIMM.Socket.A7
+    label: DDR4 DIMM
+  - name: DIMM A8
+    position: DIMM.Socket.A8
+    label: DDR4 DIMM
+  - name: DIMM B1
+    position: DIMM.Socket.B1
+    label: DDR4 DIMM
+  - name: DIMM B2
+    position: DIMM.Socket.B2
+    label: DDR4 DIMM
+  - name: DIMM B3
+    position: DIMM.Socket.B3
+    label: DDR4 DIMM
+  - name: DIMM B4
+    position: DIMM.Socket.B4
+    label: DDR4 DIMM
+  - name: DIMM B5
+    position: DIMM.Socket.B5
+    label: DDR4 DIMM
+  - name: DIMM B6
+    position: DIMM.Socket.B6
+    label: DDR4 DIMM
+  - name: DIMM B7
+    position: DIMM.Socket.B7
+    label: DDR4 DIMM
+  - name: DIMM B8
+    position: DIMM.Socket.B8
+    label: DDR4 DIMM
```

Comment:
Unfortunately this is not suppose to be modelled in NetBox at the moment. Please remove these definitions.